src/main/java/uk/gov/companieshouse/acsp/models/dao/AcspDataDao.java

```diff
@@ -54,6 +52,9 @@ public class AcspDataDao {
     @JsonProperty("how_are_you_registered_with_aml")
     private String howAreYouRegisteredWithAml;
 
+    @JsonProperty("acsp_type")
+    private String acspType;
```

Comment:
Would this not be best as an enum value?

Reply:
Enum added now